EqualityAsserts.cs

```diff
@@ -9,6 +9,10 @@
 using System.Collections.Generic;
 using System.Globalization;
 using Xunit.Sdk;
+#if XUNIT_ASSERT_ARRAY_AS_SPAN
+using System.Linq;
+using static System.MemoryExtensions;
```

Comment:
I couldn't find any reason for this line. The code compiles without it.

Reply:
You're absolutely right. Sorry about that.